redhat-developer/app-services-cli
# feat(kafka): add "kafka update" command

Closes https://github.com/redhat-developer/app-services-cli/issues/844

This adds the ability to change the owner of a Kafka instance. There will need to be a follow-up PR to add completions and loading of existing users in organization for validation purposes but this is not possible yet.

### Verification Steps

1. Run `rhoas kafka update --owner "other-user-in-your-org"`
2. Run `rhoas kafka describe` to verify that the "owner" field is changed.

### Type of change


### Checklist
